tests/test_group.py

```diff
@@ -4,8 +4,8 @@
 
 from random import randrange
 
+import uuid
```

Comment:
Nit: Bump this up one more line so there's a blank line between system and third-party imports.